ffwd-http-reporter/src/test/java/com/spotify/metrics/ffwd/FastForwardHttpReporterTest.java

```diff
@@ -218,4 +212,132 @@ public void shouldAddExtractedTags() throws Exception {
+            of("unit", "spec/s", "stat", "1m", "metric_type", "meter"), RESOURCE, VALUE_0, TIME));
+        expected.add(new Batch.Point("prefix.meter",
+            of("unit", "spec/s", "stat", "5m", "metric_type", "meter"), RESOURCE, VALUE_0, TIME));
+        expected.add(new Batch.Point("prefix.meter2",
+            of("unit", "n", "stat", "count", "metric_type", "meter"), RESOURCE, VALUE_0, TIME));
+        expected.add(new Batch.Point("prefix.meter2",
+            of("unit", "n/s", "stat", "1m", "metric_type", "meter"), RESOURCE, VALUE_0, TIME));
+        expected.add(new Batch.Point("prefix.meter2",
+            of("unit", "n/s", "stat", "5m", "metric_type", "meter"), RESOURCE, VALUE_0, TIME));
+        expected.add(
+            new Batch.Point("prefix.timer", of("unit", "ns", "stat", "max", "metric_type", "timer"),
+                RESOURCE, VALUE_0, TIME));
+        expected.add(
+            new Batch.Point("prefix.timer", of("unit", "ns", "stat", "min", "metric_type", "timer"),
+                RESOURCE, VALUE_0, TIME));
+        expected.add(new Batch.Point("prefix.timer",
+            of("unit", "ns", "stat", "mean", "metric_type", "timer"), RESOURCE, VALUE_0, TIME));
+        expected.add(
+            new Batch.Point("prefix.timer", of("unit", "ns", "stat", "p75", "metric_type", "timer"),
+                RESOURCE, VALUE_0, TIME));
+        expected.add(new Batch.Point("prefix.timer",
+            of("unit", "ns", "stat", "median", "metric_type", "timer"), RESOURCE, VALUE_0, TIME));
+        expected.add(new Batch.Point("prefix.timer",
+            of("unit", "ns", "stat", "stddev", "metric_type", "timer"), RESOURCE, VALUE_0, TIME));
+        expected.add(
+            new Batch.Point("prefix.timer", of("unit", "ns", "stat", "p99", "metric_type", "timer"),
+                RESOURCE, VALUE_0, TIME));
+        expected.add(new Batch.Point("prefix.timer",
+            of("unit", "ns/s", "stat", "1m", "metric_type", "timer"), RESOURCE, VALUE_0, TIME));
+        expected.add(new Batch.Point("prefix.timer",
+            of("unit", "ns/s", "stat", "5m", "metric_type", "timer"), RESOURCE, VALUE_0, TIME));
+        expected.add(new Batch.Point("prefix.gauge",
+            of("what", "some-gauge", "unit", "n", "metric_type", "gauge"), RESOURCE, VALUE_0,
+            TIME));
+
+        reporter.start();
+
+        final ArgumentCaptor<Batch> batch = ArgumentCaptor.forClass(Batch.class);
+
+        reporter.stopWithFlush();
+
+        verify(httpClient, atLeastOnce()).sendBatch(
+            batch.capture());
+        for (final Batch b : batch.getAllValues()) {
+            assertEquals(commonTags, b.getCommonTags());
+            final Set<Batch.Point> points = new HashSet<>(b.getPoints());
+            points.removeAll(expected);
+            assertEquals("expected empty set of points", ImmutableSet.of(), points);
+        }
+    }
+
+    @Test
+    public void someReportingNoTickFails() {
```

Comment:
I'm not sure best description is that it fails? Perhaps a more descriptive name could be `noReportingIfNoTick`?